src/panels/lovelace/cards/hui-picture-elements-card.ts

```diff
@@ -0,0 +1,103 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import createHuiElement from "../common/create-hui-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types";
+import { HomeAssistant } from "../../../types.js";
+import { ElementConfig } from "../elements/types.js";
+import { TemplateResult } from "lit-html";
+
+interface Config extends LovelaceConfig {
+  title?: string;
+  image: string;
+  elements: ElementConfig[];
+}
+
+class HuiPictureElementsCard extends LitElement implements LovelaceCard {
+  protected config?: Config;
+  private _hass?: HomeAssistant;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  set hass(hass: HomeAssistant) {
+    this._hass = hass;
+    for (const el of this.shadowRoot!.querySelectorAll("#root > *")) {
+      const element = el as LovelaceCard;
+      element.hass = this._hass;
+    }
+  }
+
+  public getCardSize(): number {
+    return 4;
+  }
+
+  public setConfig(config: Config): void {
+    if (!config) {
+      throw new Error("Invalid Configuration");
+    } else if (!config.image) {
+      throw new Error("Invalid Configuration: image required");
+    } else if (!Array.isArray(config.elements)) {
+      throw new Error("Invalid Configuration: elements required");
+    }
+
+    this.config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this.config || !this.config.image || !this.config.elements) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <ha-card .header="${this.config.title}">
+        <img src="${this.config.image}">
```

Comment:
just make this `image!`

Reply:
Also, the img tag used to be _inside_ the root div. I think that's why it's not rendering.

Reply:
Yep, confirmed it. Moving the img into the root div with chrome dev tools on the gallery solves the elements not being visible.

Reply:
I moved it outside in haste because I didn't want to assign `hass` to it during `set hass` where I grab all elements inside `root`.

Did `more-info` work for you? It is not working for me now that I have the elements rendering.

Reply:
I don't follow the `image!` comment 😕

Reply:
`more-info` works for me.

Reply:
I think he may mean like this: https://github.com/home-assistant/home-assistant-polymer/blob/34bfc1264708b9b60dbd47084b19f2d077dd95e9/src/panels/lovelace/cards/hui-picture-entity-card.js#L55

Reply:
The reason I referred to making it `image!` is because you are checking above if image is set because probably typescript complained about this line being used without checking if image existed. By using `!` at the end, you can ignore that typescript warning.